### PR TITLE
Add description of external role

### PR DESCRIPTION
**Pull request by astrozzc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,27 @@ We added support for the new role flag "admin_default", similar to “platform_d
 }
 ```
 
+External Role
+---------------------
+We added support to define external role. In the `roles` directory, the external role could be defined as a regular role but without access/permission. The external role should have a field `external` defined, which has external id and external tenant. An example ocm role is displayed below.
```
Comment (coderbydesign) [inline README.md:135]:
```suggestion
We added support to define external roles. In the `roles` directory, an external role can be defined as a regular role but without access/permissions. An external should have an object `external` defined, which has external `id` and external `tenant` fields. An example ocm role is displayed below.
```

File: README.md [hunk 2]
```diff
@@ -130,6 +130,27 @@ We added support for the new role flag "admin_default", similar to “platform_d
 }
 ```
 
+External Role
```
Comment (coderbydesign) [inline README.md:133]:
```suggestion
External Roles
```

File: README.md [hunk 3]
```diff
@@ -130,6 +130,27 @@ We added support for the new role flag "admin_default", similar to “platform_d
...
+      "version": 2,
+      "external": {
+        "id": "ClusterOwner",
+        "tenant": "ocm"
+      }
+    }
+  ]
+}
+```
+After the role is seeded into system. The role could be assigned to customer through group and the external team could use the external id of the rbac role to identify it as one of the role on their side.
```
Comment (coderbydesign) [inline README.md:152]:
```suggestion
After the role is seeded into system, the role can be assigned to users through groups, and the external tenant team can use the external id of the RBAC role to identify it and map it to one of their roles.
```

**Review by coderbydesign — COMMENTED** [review]
Hey @astrozzc! Just had a few wording suggestions, but was also wondering if we could add something to our [role schema](https://github.com/RedHatInsights/rbac-config/blob/master/schemas/roles.schema) to validate the fields on `external` when it's present?

**Review by coderbydesign — APPROVED** [review]

**Comment by astrozzc** [comment]
Thanks for the suggestions!

**Pull request merged**